src/components/PasswordField.jsx

```diff
@@ -40,6 +67,11 @@ const PasswordGeneratorField = (props) => {
           onClick={() => handlePasswordChange(pwg.getRandomPassword())}>
           <LockReset />
         </IconButton>
+        <IconButton
+          aria-label="compromised password check"
+          onClick={() => onCompromisedPasswordCheck(password)}>
```

Comment:
`password` is a prop being passed in to the main function so we don't need to pass it to `onCompromisedPasswordCheck`. However is it safer in the long run to do so?

Reply:
I considered not passing it to the function but I thought it would be better in case the function got refactored out of the component later.